src/components/genreSelect/genreSelect.tsx

```diff
@@ -0,0 +1,13 @@
+import './genreSelect.css';
+const GenreSelect = ({genreList, selectedGenre, onSelect}: GenreSelectProps)=> {
+return (
+    <ul className="genreList">
+        {genreList.map((genre) => {
+            return(
+                <li className="listItem" key={genre} onClick={onSelect(genre)}><a href='#' className={genre === selectedGenre ? "selectedItem" : ""}>{genre}</a></li>
```

Comment:
Please configure the code formation for your project - to improve readability and follow best practices.

Reply:
The className needs to be more descriptive. 'listItem' => 'genreListItem'. Or smth like this.